plugins/st2/st2.go

```diff
@@ -0,0 +1,25 @@
+package st2
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/needsauth"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+)
+
+func StackStormCLI() schema.Executable {
+	return schema.Executable{
+		Name:      "StackStorm st2 CLI",
+		Runs:      []string{"st2"},
+		DocsURL:   sdk.URL("https://docs.stackstorm.com/reference/cli.html"),
+		NeedsAuth: needsauth.IfAll(
+			needsauth.NotForHelpOrVersion(),
+			needsauth.NotWithoutArgs(),
+		),
```

Comment:
I think we should also circumvent auth in case `--config-file` is passed.
```
st2 --config-file=~/.st2/prod-config action list
```
is the example given in the docs.